### PR TITLE
use top-nav on desktop

### PR DESCRIPTION
**Pull request by andrewdamelio**
Screen Shot 2019-07-30 at 4 31 56 PM
Screen Shot 2019-07-30 at 4 32 17 PM

### CODE REVIEW

**Comment by bthaile** [comment]
if user nav to market trading page they can't click the `<---` and go back to market listing page. 
This can be a follow up tix, just create one if not added to this PR

**Comment by bthaile** [comment]
we can address reporting during reporting/disputing redesign effort.

image

File: packages/augur-ui/src/modules/app/components/app.tsx [hunk 1]
```diff
@@ -534,18 +539,23 @@ export default class AppView extends Component<AppProps, AppState> {
...
             role="presentation"
           >
-            <div className={Styles.Logo}>
-              <Link to={makePath(DEFAULT_VIEW)}>
-                <Logo />
-              </Link>
-            </div>
             {this.renderMobileMenuButton(unseenCount)}
-            <SideNav
+            {isMobile && <SideNav
```
Comment (bthaile) [inline packages/augur-ui/src/modules/app/components/app.tsx]: we are getting away from using `isMobile` is there a different way?

File: packages/augur-ui/src/modules/app/components/app.tsx [hunk 2]
```diff
@@ -555,19 +565,16 @@ export default class AppView extends Component<AppProps, AppState> {
                 sidebarStatus.mobileMenuState ===
                 MOBILE_MENU_STATES.SIDEBAR_OPEN
               }
-              menuScalar={subMenu.scalar}
               menuData={this.sideNavMenuData}
               currentBasePath={sidebarStatus.currentBasePath}
-            />
+            />}
+
+            {!isMobile && <TopNav
```
Comment (bthaile) [inline packages/augur-ui/src/modules/app/components/app.tsx]: same comment as above

File: packages/augur-ui/src/modules/app/components/top-nav/top-nav.styles.less [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+        color: @color-tertiary-info-text;
+        text-transform: capitalize;
+      }
+    }
+  }
+}
+
+.Selected {
+  border-bottom: @size-1 solid @color-secondary-text !important;
```
Comment (bthaile) [inline packages/augur-ui/src/modules/app/components/top-nav/top-nav.styles.less]: any way to not use `!important`

File: packages/augur-ui/src/modules/app/components/top-nav/top-nav.styles.less [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+  > a > span {
+    color: @color-secondary-text !important;
```
Comment (bthaile) [inline packages/augur-ui/src/modules/app/components/top-nav/top-nav.styles.less]: any way to not use `!important`

File: packages/augur-ui/src/modules/app/components/top-nav/top-nav.styles.less [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+  }
+}
+
+.CreatetButton {
+  margin-left: @size-24;
+
+  button {
+    padding: @size-9 @size-16 !important;
```
Comment (bthaile) [inline packages/augur-ui/src/modules/app/components/top-nav/top-nav.styles.less]: any way to not use `!important`

**Review by bthaile — COMMENTED** [review]
a few comments about standards / convention directions

File: packages/augur-ui/src/modules/app/components/app.tsx (hunk 1 above)
Comment (andrewdamelio) [reply packages/augur-ui/src/modules/app/components/app.tsx]: Yup - fixing

**Review by bthaile — APPROVED** [review]
lgtm, nice to clean up some `isMobile` as well.

**Pull request merged**